lib/core/versions/latest/InputValidator.ts

```diff
@@ -51,12 +51,10 @@ export default class InputValidator {
       );
     }
 
-    const casFileUriAsHashBuffer = Encoder.decodeAsBuffer(casFileUri);
-    const hashAlgorithmInMultihashCode = ProtocolParameters.hashAlgorithmInMultihashCode;
-    if (!Multihash.isComputedUsingHashAlgorithm(casFileUriAsHashBuffer, hashAlgorithmInMultihashCode)) {
+    if (casFileUri.length > ProtocolParameters.maxCasUriLength) {
       throw new SidetreeError(
-        ErrorCode.InputValidatorCasFileUriUnsupported,
```

Comment:
Sanity: Remove in `ErrorCode` if `InputValidatorCasFileUriUnsupported` no longer referenced.